### PR TITLE
NPM script directive resort to the system terminal to run the command…

### PR DESCRIPTION
**Pull request by dmarcos**
…s. cmd.exe in windows and sh, bash... in OSX and

Unix based system. Bash sell commands don't work on Windows. If we want to have a cross platform build system we have to
define our directives using pure Node scripts. This patch replaces current commands with equivalent ones that rely on
node modules with identical functionality: mkdirp, ncp, open...

### CODE REVIEW

**Comment by dmarcos** [comment]
@cvan what do you think?
I have removed standard-reporter because it doesn't seem to work on Windows. It just doesn't give any output. I guess it's just failing silently.  
The same for the gh-pages module. gh-pages.js command finishes execution but nothing is pushed to the gh-pages branch of the remote.

File: package.json [hunk 1]
```diff
@@ -32,21 +33,24 @@
...
-    "standard-reporter": "^1.0.5",
     "uglifyjs": "^2.4.10"
   },
   "scripts": {
     "start": "npm run dev",
     "dev": "npm run build && node ./scripts/budo",
     "browserify": "browserify ./src/index.js -s aframe-core",
-    "build": "mkdir -p build/ && npm run browserify -- --debug -o build/aframe-core.js",
-    "dist": "rm -rf dist/ && mkdir -p dist/ && npm run dist:js",
+    "build": "mkdirp build/ && npm run browserify -- --debug -o build/aframe-core.js",
```
Comment (cvan) [inline package.json]: I would recommend using shelljs instead

Comment (dmarcos) [inline package.json]: @cvan Just looked at (shelljs)[https://github.com/shelljs/shelljs]. How can you use directly from the command line/script directive without resorting to an external script? shelljs installs a ```shjs``` binary that takes a script file but I don't see any example that lets me do something like this: shjs "rm -rf dist/"

**Comment by dmarcos** [comment]
I'm going to land this to fix the Windows situation. I'm open to rewrite the npm directives in ```shelljs``` in a follow up patch.

**Pull request merged**